ClassPaulita/GuessTheCardGame.cs

```diff
@@ -26,39 +160,60 @@ private bool UserChoseToQuit(char userChoice)
             return userChoice == 'q' || userChoice == 'Q';
         }
 
-        private bool IsUserChoiceValid(char userChoice)
-        {                      
+        private char GetUserGuess(Card previousCard)
+        {
+            Console.Write($"\n\n\tIs the next card greater than {previousCard}(y or n)? ");
+            char userGuess = Convert.ToChar(Console.ReadLine());
 
-            if (UserChoseToPlay(userChoice) || UserchoseToSeeInstructions(userChoice) || UserChosetoSeeBoard(userChoice) || UserChoseToQuit(userChoice))
+            while (!IsUserGuessValid(userGuess))
             {
-                return true;
+                Console.WriteLine("\n\tYour answer is not valid. Please try again!");
+                Console.Write($"\n\n\tIs the next card higher or lower than {previousCard}(y or n)? ");
+                userGuess = Convert.ToChar(Console.ReadLine());
             }
 
-            return false;
+            return userGuess;
         }
 
-        private char GetUserMenuChoice()
+        private void AddUserScoreToBoard(int userScore)
         {
+            Console.WriteLine($"Congratulations! You made it to the Leader Board.\n{_spacing}Please enter your userName (10 characters max):\n\n{_spacing}");
+            string userName = Console.ReadLine();
 
-            string menu = $"{spacing}Press P To Play" +
-                          $"{spacing}Press I For Instructions" +
-                          $"{spacing}Press B To See The High Score Board" +
-                          $"{spacing}Press Q To Quit";
+            _scoreBoard.Add(new KeyValuePair<int, string>(userScore, userName));
```

Comment:
You add this to the score board before checking if it's valid???

Reply:
k. I messed up the order of the lines

Reply:
I'm not seeing the fix, but you marked this as resolved

Reply:
I see it now, never mind. Nice job